docs/deploying-kas-fleet-manager-to-openshift.md

```diff
@@ -146,6 +146,7 @@ make deploy/service IMAGE_TAG=<your-image-tag-here> <OPTIONAL_PARAMETERS>
 - `MAS_SSO_ENABLE_AUTH`: Enables MAS SSO authentication for the Data Plane. Defaults to `true`.
 - `MAS_SSO_BASE_URL`: MAS SSO base url. Defaults to `https://identity.api.stage.openshift.com`.
 - `MAS_SSO_REALM`: MAS SSO realm url. Defaults to `rhoas`.
+- `SPECIAL_MANAGEMENT_ORG_ID`: Special Management Organization ID used for creating internal Service accounts. Defaults to the Org ID of stage.
```

Comment:
```suggestion
- `SPECIAL_MANAGEMENT_ORG_ID`: Special Management Organization ID used for creating internal Service accounts. Defaults to `13640203` which is the special management organisation id for Stage environment.
```

Reply:
Cannot expose the Org ID in a public repo. this is the Org ID that will have all the internal service accounts, so better not to specify the same in ReadME. @akoserwal i forgot that this is a public repo, so should we think about defaulting to stage OrgID - const declared in the code? Probably we can get away with an empty string. Would prefer not to have this in the public domain

Reply:
agreed. code changed